src/connectors/autocomplete/connectAutocomplete.ts

```diff
@@ -35,32 +22,46 @@ type AutocompleteConnectorParams = {
   escapeHTML?: boolean;
 };
 
-export type AutocompleteRendererOptions<TAutocompleteWidgetParams> = {
+export type AutocompleteRendererOptions = {
+  /**
+   * The current value of the query.
+   */
   currentRefinement: string;
-  indices: AutocompleteIndex[];
+
+  /**
+   * The indices this widget has access to.
+   */
+  indices: Array<{
+    /**
+     * The name of the index
+     */
+    indexName: string;
+
+    /**
+     * The resolved hits from the index matching the query.
+     */
+    hits: Hits;
+
+    /**
+     * The full results object from the Algolia API.
+     */
+    results: SearchResults;
+  }>;
+
+  /**
+   * The current instant search instance.
+   *
+   * @internal
+   */
   instantSearchInstance: InstantSearch;
```

Comment:
This should already be handled by the `RendererOptions` type: https://github.com/algolia/instantsearch.js/blob/ad1b5c6e912bf41026fae55fb451c2e4cb28f36a/src/types/connector.ts#L20